src/tests/worker/test_suggestion_processor.py

```
@@ -0,0 +1,352 @@
+        translations=[
+            SimpleNamespace(verse_id="not-a-valid-id", target_text="bad"),
+            SimpleNamespace(verse_id="MAT_1_1", target_text="good translation"),
+        ]
+    )
+
+    import httpx
+
+    posted_payloads = []
+
+    class _FakeContextResponse:
+        def raise_for_status(self):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `raise_for_status`

Add return type annotation: `None`

(ANN202)